### PR TITLE
Update KEEP token dashboard for subscribing to `Staked` event

### PR DESCRIPTION
**Pull request by r-czajkowski**
This PR fixes:
* subscription to the `Staked` event. `Staked` event's params have changed. Currently, a `Staked` event emits: `owner, operator, beneficiary, value`, instead of `from, value`.

### CODE REVIEW

**Comment by dimpar** [comment]
Delegation and undelegation went trough

**Comment by r-czajkowski** [comment]
Converted to draft since the event subscription does not work with MetaMask 8.0.2. Works with 7.7.9. I'm trying to figure out what's wrong.

**Comment by r-czajkowski** [comment]
It seems like an error on the Metmask side https://github.com/MetaMask/metamask-extension/issues/8894. Event subscription works like a charm in `8.0.3`.

File: solidity/dashboard/src/pages/TokensPageContainer.jsx [hunk 1]
```diff
@@ -312,13 +316,19 @@ const isGranteeInManagedGrant = async (web3Context, grantee) => {
...
   // check if grantee is a contract
   const code = await web3.eth.getCode(grantee)
+  console.log("codee code", code)
+
   if (!isCodeValid(code)) {
+    console.log("codee return false", code)
+
     return false
   }
+  console.log("codee code", code)
```
Comment (dimpar) [inline solidity/dashboard/src/pages/TokensPageContainer.jsx:326]: codee code

File: solidity/dashboard/src/pages/TokensPageContainer.jsx [hunk 2]
```diff
@@ -301,13 +316,19 @@ const isGranteeInManagedGrant = async (web3Context, grantee) => {
...
 
   const granteeAddressInManagedGrant = await managedGrantContractInstance.methods
     .grantee()
     .call()
+  console.log("granteeAddressInManagedGrant", granteeAddressInManagedGrant)
```
Comment (dimpar) [inline solidity/dashboard/src/pages/TokensPageContainer.jsx:331]: console.log

File: solidity/dashboard/src/pages/TokensPageContainer.jsx [hunk 3]
```diff
@@ -301,13 +316,19 @@ const isGranteeInManagedGrant = async (web3Context, grantee) => {
 
   // check if grantee is a contract
   const code = await web3.eth.getCode(grantee)
+  console.log("codee code", code)
+
   if (!isCodeValid(code)) {
+    console.log("codee return false", code)
```
Comment (dimpar) [inline solidity/dashboard/src/pages/TokensPageContainer.jsx:322]: codee code

File: solidity/dashboard/src/pages/TokensPageContainer.jsx [hunk 4]
```diff
@@ -301,13 +316,19 @@ const isGranteeInManagedGrant = async (web3Context, grantee) => {
 
   // check if grantee is a contract
   const code = await web3.eth.getCode(grantee)
+  console.log("codee code", code)
```
Comment (dimpar) [inline solidity/dashboard/src/pages/TokensPageContainer.jsx:319]: codee code

**Pull request merged**